composer.json

```diff
@@ -32,5 +32,7 @@
             "dev-master": "3.0-dev"
         }
     },
+    "minimum-stability": "dev",
```

Comment:
hmmm - this means we're using 2.0-dev right now, but as soon as 1.20.0 is tagged, composer will use that in the lock file for this repo